tests/datasource/test_new_datasource_with_sql_data_connector.py

```diff
@@ -60,6 +60,8 @@ def test_basic_instantiation(sa):
     report = my_data_connector.self_check()
     print(json.dumps(report, indent=4))
 
+    report["execution_engine"].pop("connection_string")
```

Comment:
just curious, does the following assertion fail with the `connection_string` still in it?

Reply:
you answered in the PR description :) resolved

Reply:
@Shinnnyshinshin Yes, because I modified the config of the execution engine to NOT suppress this value in its diagnostic (it can be useful for troubleshooting, but not very useful for the unit test).